modules/users/client/views/invite.client.view.html

```diff
@@ -15,10 +15,10 @@
 
         <p class="lead">Your invitation code</p>
 
-        <input type="text"
+        <input id="invite" type="text"
```

Comment:
Move `type` to newline to align with others.

id could be something less generic, eg. `tr-invite-code`?